internal/js/modules/k6/browser/browser/registry.go

```diff
@@ -184,6 +184,8 @@ type browserRegistry struct {
 
 type browserBuildFunc func(ctx, vuCtx context.Context) (*common.Browser, error)
 
+var fidDeprecationWarningOnce sync.Once
```

Comment:
I'm not entirely happy with this approach. Is there something I'm missing, maybe a state that I can rely on that allows me to log a single warning log regardless of how many iterations, VUs and scenarios are in the test?